doc/introduction/remote.rst

```diff
@@ -352,52 +296,101 @@ Rgate(1.902) | (q[0])
 Rgate(-1.173) | (q[5])
 MeasureFock | (q[0], q[1], q[2], q[3], q[4], q[5], q[6], q[7])
 
-The compiled program can be saved as a new Blackbird script using the :func:`~.io.save` function:
+While equivalent to the uncompiled program, we can now see the low-level hardware
+operations that are applied on the physical chip.
 
->>> from strawberryfields.io import save
->>> save("test_compiled.xbb", prog)
 
+Working with Blackbird scripts
+------------------------------
 
-Tips and tricks
----------------
+When submitting quantum programs to be executed remotely, they are communicated to
+the cloud platform using Blackbird --- a quantum photonic assembly language.
+Strawberry Fields also supports exporting programs directly as Blackbird scripts
+(an ``xbb`` file); Blackbird scripts can then be submitted to be executed via the
+Strawberry Fields :doc:`command line interface </code/sf_cli>`.
```

Comment:
Does this link render properly? (didn't check myself, but looks similar to ones @trbromley commented on)

Reply:
Yep! 
![image](https://user-images.githubusercontent.com/2959003/77225480-67a4ca80-6bbf-11ea-8827-a1df10ae7743.png)